lib/auth/auth.go

```diff
@@ -1394,6 +1394,7 @@ func (a *Server) generateUserCert(req certRequest) (*proto.Certs, error) {
 		Username:          req.user.GetName(),
 		Impersonator:      req.impersonator,
 		Groups:            req.checker.RoleNames(),
+		AccessPolicies:    req.user.GetAccessPolicies(),
```

Comment:
this adds them to the tls cert, what about the SSH cert (above)? Would probably be good to add a test that the policies are correctly encoded in both cert types on login

Reply:
good catch! I fixed this and added a test.